pkg/cri/sbserver/image_pull.go

```diff
@@ -356,6 +372,8 @@ func hostDirFromRoots(roots []string) func(string) (string, error) {
 
 // registryHosts is the registry hosts to be used by the resolver.
 func (c *criService) registryHosts(ctx context.Context, auth *runtime.AuthConfig, updateClientFn config.UpdateClientFunc) docker.RegistryHosts {
+	ctx, span := tracing.StartSpan(ctx, makeSpanName("registryHosts"))
```

Comment:
I'm hesitant to expose Go's unexported methods to spans. Wouldn't people treat them as a part of the published interface?

Reply:
I am not sure I agree to that. Isn't the idea of tracing  being able to look into the unexposed methods, say a method that does network calls to get more visibility into the network latency?

For this case though, I don't think there is any real value to instrumenting this method so I will go ahead and remove the instrumentation.

Reply:
I think there can be value for long operations like network, but not for small functions like this.

Reply:
Updated PR to remove this operation